code/__DEFINES/mobs.dm

```diff
@@ -54,6 +54,14 @@
 #define BODYPART_ORGANIC   1
 #define BODYPART_ROBOTIC   2
 
+//Reagent Metabolization flags, defines the type of reagents that affect this mob
+#define PROCESS_ORGANIC 1		//Only processes reagents with "ORGANIC" or "ORGANIC | SYNTHETIC"
+#define PROCESS_SYNTHETIC 2		//Only processes reagents with "SYNTHETIC" or "ORGANIC | SYNTHETIC"
+
+// Reagent type flags, defines the types of mobs this reagent will affect
+#define ORGANIC 1
+#define SYNTHETIC 2
```

Comment:
if you can, please try to use mob_biotypes - species can already set biotypes, and doing it that way will help pave the way to refactoring chems to living instead of carbon.

Reply:
Currently working on this as of (2:42PM EST, 6/16/2018)